src/ci/radamsa-windows.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# NOTE: This assumes building within msys64/bash
+
+set -ex
+
+export PATH=/usr/bin:$PATH
+pacman -S gcc gzip --noconfirm
+
+mkdir -p artifacts/third-party/radamsa-win64
+git clone https://gitlab.com/akihe/radamsa
+cd radamsa
+git checkout 8121b78fb8f87e869cbeca931964df2b32435eb7
```

Comment:
Maybe worth just flagging what this commit is expected to correspond to, if there's anything important in that choice. Looks like a snapshot of latest `develop` at time of writing?